rockstar/rockstar.js

```diff
@@ -339,6 +341,38 @@
         });
     }
     
+    function directoryGroup() {
+        var groupId = location.pathname.split("/")[3];
+        var group;
+        getJSON(`/api/v1/groups/${groupId}`).then(aGroup => {
+            group = aGroup;
+            $(".subheader").html(`${e(group.profile.name)}`);
+            document.title += ` - ${e(group.profile.name)} ${e(group.profile.description)}`;
+        });
+            function showGroup() {
+                function toString(o, i = '') {

```

Comment:
this code for `toString` was copied and pasted from showUser. it should be factored out.

also, indentation should be cleaned up